src/tools/rbd/CMakeLists.txt

```diff
@@ -68,8 +68,6 @@ target_link_libraries(rbd
   rbd_types
   librbd
   journal
-  libneorados
-  librados
```

Comment:
Is the idea that `rbd` target would be getting these transitively from `cls_rbd_client` target?

This doesn't seem right to me because the CLI tool which is handled by this target uses both librados and librbd -- at the very least it needs to connect to the cluster and hence call methods like `init_with_context()` and `connect()` on `librados::Rados` object before doing anything specific to RBD.